core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/AdminApiServiceSpec.scala

```diff
@@ -92,12 +93,21 @@ class AdminApiServiceSpec extends ApiServiceSpec {
       ActiveSubmission(constantData.workspace.namespace, constantData.workspace.name, constantData.submission1),
       ActiveSubmission(constantData.workspace.namespace, constantData.workspace.name, constantData.submission2))
 
-    Get(s"/admin/submissions") ~>
-      sealRoute(services.adminRoutes) ~>
-      check {
-        assertResult(StatusCodes.OK) { status }
-        assertSameElements(expected, responseAs[Seq[ActiveSubmission]])
+    withStatsD {
+      Get("/admin/submissions") ~>
+        sealRoute(instrumentRequest { services.adminRoutes }) ~>
+        check {
+          assertResult(StatusCodes.OK) {
+            status
+          }
+          assertSameElements(expected, responseAs[Seq[ActiveSubmission]])
+        }
+    } { capturedMetrics =>
+      val expected = expectedHttpRequestMetrics("get", "admin.submissions", StatusCodes.OK.intValue, 1)
+      assert {
```

Comment:
OK, I added `RawlsTestUtils.assertSubsetOf()` which uses `should contain` internally